wrappers/android/librealsense/src/main/java/com/intel/realsense/librealsense/Sensor.java

```diff
@@ -21,10 +21,16 @@
     public <T extends Sensor> T as(Extension extension) {
         switch (extension){
             case ROI: return (T) new RoiSensor(mHandle);
+            case DEPTH_SENSOR: return (T) new DepthSensor(mHandle);
+            case COLOR_SENSOR: return (T) new ColorSensor(mHandle);
```

Comment:
Pls refactor "throw new RuntimeException("this sensor is not extendab..." under  `default` case